README.md

```diff
@@ -65,6 +66,7 @@ Merci à tous les contributeurs 🙏
     <td align="center"><a href="https://github.com/Nabelle"><img src="https://avatars.githubusercontent.com/u/41482051?v=4" width="100px;" alt=""/><br /><sub><b>Nabelle</b></sub></a></td>
     <td align="center"><a href="https://github.com/jeanphibaconnais"><img src="https://avatars.githubusercontent.com/u/32639372?v=4" width="100px;" alt=""/><br /><sub><b>Jean-Phi Baconnais</b></sub></a></td>
     <td align="center"><a href="https://gerome.dev"><img src="https://avatars.githubusercontent.com/u/32737308?v=4" width="100px;" alt=""/><br /><sub><b>Gérôme Grignon</b></sub></a></td>
+    <td align="center"><a href="https://github.com/iGranDav"><img src="https://avatars.githubusercontent.com/u/75418?v=4" width="100px;" alt=""/><br /><sub><b>David Bonnet</b></sub></a></td>
```

Comment:
This is made by the bot which manage contributeur :)

Reply:
nice 🙂